frontend/app/views/info/feedback.scala.html

```diff
@@ -6,27 +6,41 @@
 
     <main role="main" class="page-content">
 
-        @fragments.page.pageHeader("Feedback", Some("Help us improve your membership experience"))
+        <section class="form-header">
```

Comment:
Is a pageHeader displayed differently to a form header? If so we might want to make this a fragment in the same style.